src/main/java/app/freerouting/gui/BoardToolbar.java

```diff
@@ -8,27 +8,27 @@
  */
 class BoardToolbar extends javax.swing.JPanel
 {
-    
+
     /** Creates a new instance of BoardToolbarPanel */
     BoardToolbar(BoardFrame p_board_frame)
     {
         this.board_frame = p_board_frame;
-        
-        java.util.ResourceBundle resources = 
+
+        java.util.ResourceBundle resources =
                 java.util.ResourceBundle.getBundle("app.freerouting.gui.BoardToolbar", p_board_frame.get_locale());
-        
+
         this.setLayout(new java.awt.BorderLayout());
-        
+
         // create the left toolbar
-        
+
         final javax.swing.JToolBar left_toolbar = new javax.swing.JToolBar();
         final javax.swing.ButtonGroup toolbar_button_group = new javax.swing.ButtonGroup();
         this.select_button = new javax.swing.JToggleButton();
         this.route_button = new javax.swing.JToggleButton();
         this.drag_button = new javax.swing.JToggleButton();
         final javax.swing.JLabel jLabel1 = new javax.swing.JLabel();
-        
-        left_toolbar.setMaximumSize(new java.awt.Dimension(1200, 23));
+
+        left_toolbar.setMaximumSize(new java.awt.Dimension(1200, 30));
```

Comment:
Do you know why we had the problem with this? I mean why is that I don't have it on Windows, but there is a problem on Linux? Can this be somehow related to the DPI/resolution settings? 
I'm not sure, because on Windows I changed the "Scale" setting, and it didn't make a difference, it looked good with all settings.

Reply:
Do you have a picture comparing both changes to show me?

Reply:
Here might be the explanation
https://stackoverflow.com/questions/65524291/cannot-set-maximum-size-for-jframe-under-windows

Reply:
Hmm, that might be it! 
Alright then, if I understand it correctly that means that the changes you made are necessary and they work as expected on Linux and they don't affect Windows users. 👍